gettsim/transfers/rente.py

```diff
@@ -274,25 +300,419 @@ def ges_rente_zugangsfaktor(
+            thresholds=ges_rente_params["altersrente_für_frauen"]["thresholds"],
+            rates=ges_rente_params["altersrente_für_frauen"]["rates"],
+            intercepts_at_lower_thresholds=ges_rente_params["altersrente_für_frauen"][
+                "intercepts_at_lower_thresholds"
+            ],
+        )
+    else:
+        pension_for_women = (
+            9000  # arbitrary high number to represent non-feasible retirement age
+        )
+
+    out = min(pension_for_women, pension_for_long)
+
+    return out
+
+
+def ges_rente_grenz_voll_altersrente(
+    geburtsjahr: int,
+    ges_rente_params: dict,
+    ges_rente_grenze_langjährig_frau: float,
+    ges_rente_regelaltersgrenze: float,
+    ges_rente_vorrauss_besond_lang: bool,
+) -> float:
+    """Calculates the age, at which a person is eligible to claim the full pension.
+        Full retirement age (FRA) without deductions. This age is smaller or equal
+        to the regelaltersgrenze (FRA<=NRA) and depends on personal characteristics
+        as gender, insurance duration, health/disability, employment status.
 
     Parameters
     ----------
     geburtsjahr
         See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
     ges_rente_params
         See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+    ges_rente_grenze_langjährig_frau
+        See :func:`ges_rente_grenze_langjährig_frau`.
+    ges_rente_regelaltersgrenze
+        See :func:`ges_rente_regelaltersgrenze`.
+    ges_rente_vorrauss_besond_lang
+        See:func:`ges_rente_vorrauss_besond_lang`.
+
+    Returns
+    -------
+    Lowest possible full retirement age (without deductions).
+    """
+
+    regelrente = ges_rente_regelaltersgrenze
+    pension_long_or_women = ges_rente_grenze_langjährig_frau
+
+    if ges_rente_vorrauss_besond_lang:
+        pension_for_extralong = piecewise_polynomial(
+            x=geburtsjahr,
+            thresholds=ges_rente_params[
+                "altersgrenze_besonders_langjährig_versicherte"
+            ]["thresholds"],
+            rates=ges_rente_params["altersgrenze_besonders_langjährig_versicherte"][
+                "rates"
+            ],
+            intercepts_at_lower_thresholds=ges_rente_params[
+                "altersgrenze_besonders_langjährig_versicherte"
+            ]["intercepts_at_lower_thresholds"],
+        )
+    else:
+        pension_for_extralong = (
+            9000  # arbitrary high number to represent non-feasible retirement age
+        )
+
+    out = min([regelrente, pension_long_or_women, pension_for_extralong])
+
+    return out
+
+
+def ges_rente_vorraus_regelrente(ges_rente_wartezeit_5: float) -> bool:
+    """Function determining the eligibility for the Regelaltersrente.
+
+    Parameters
+    ----------
+    ges_rente_wartezeit_5
+        See :func:`ges_rente_wartezeit_5`.
+    """
+    if ges_rente_wartezeit_5 >= 5:
```

Comment:
```suggestion
    out = ges_rente_wartezeit_5 >= 5
```
and get rid of the next three lines

Similar simplifications are possible for the next functions.

Reply:
done